### ISSUE TITLE
Post Effect in post render

### ISSUE THREAD

**Issue by LeXXik** [playcanvas/engine#2519]
I am working though a setup, where I apply a standard Post Effect outline from the examples folder. It is being applied on a fast moving object, and as a result, the outline is always late, trailing after the object. The pc.PostEffect includes a `render` method, which I assume is executed before the mesh transform is applied. But I am not sure how move it to post render? All the object's transformations are done in `.update()`. 

I am looking for an advice how to apply the effect after all the transforms have taken place to avoid ghosting. Does pc.PostEffect support this out of the box, or is there some other elegant way?

**Comment by mvaligursky** [comment]
I modified model-outline.html example to move the cube with outline fast around the circle, and all renders fine there, so that is probably a good example on how to organize the effect?

**Comment by LeXXik** [comment]
Thank you for the prompt response, guys. I think I figured out what was causing the ghosting. Originally, I couldn't follow the example as is, because if I insert a layer at index 0, I get an error. I reported it in another issue.
So, I added the layer through Editor pretty late in the stack, right before UI. This results in ghosting, and the outline is late.
I changed it to now to insert at index 1 (right after World, I assume), and the outline matches the mesh now. If I change index to 10, the outline starts to lag behind.
